tools/build_release.py

```diff
@@ -50,6 +50,8 @@ def main():
         * Upload to PyPI - you will need to type in your credentials
     * Merge the pull request to origin/master
     * Create a release on GitHub using the portion from the changelog for this release for the description
+        * Add example zip files as artifacts
+        * Update LAST_RELEASE with the version of this release (1.1.5, 1.1.6, 1.2.0)
```

Comment:
To make this clearer to newcomers:

* Add the ZIP files under `generated/examples` for each module as a release artifact.
* Update contents of LATEST_RELEASE with the version of the release being created.

This said, I see we have a VERSION file in the root as well. Do you think it would make sense to have both "version" and "latest released version" exist as metadata inside the `src/` folder?

Reply:
VERSION is generated so it should not go in src. Eventually, LATEST_RELEASE should also be generated, so it also should not go in src.

Reply:
Then shouldn't VERSION go inside generated/ ?
How do we make LATEST_RELEASE be generated?

These are things worth describing in an issue.

Reply:
It is standard for repos to have a VERSION file in the root. Ours just happens to be generated based on the VERSION files of each of the drivers, which are also generated based on the version in their metadata.

When "we" create a script to create a GitHub release, then one of the things that script can (should) do is update LATEST_RELEASE.